apps/happy-blocks/src/support-page/inline-skeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import classnames from 'classnames';
+import React from 'react';
+
+type InlineSkeletonProps = {
+	loaded: boolean;
+	hidden?: boolean;
+	large?: boolean;
+	children: React.ReactNode;
+};
+
+/**
+ * Loading skeleton that takes its sizes from the font size
+ */
+export const InlineSkeleton = ( props: InlineSkeletonProps ) => {
+	if ( props.loaded ) {
+		return <>{ props.children }</>;
+	}
+	return (
+		<span
+			className={ classnames( 'hb-support-page-inline-skeleton', {
+				'hb-support-page-inline-skeleton__hidden': props.hidden,
```

Comment:
We could do an `is-hidden` or `is-large` modifiers here instead of fully repeating the class name. More info on https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md.

Reply:
Thanks for the link!
I was trying smth like it before, but Calypso linter was complaining about not adhering to BEM.
Probably, I was doing it differently, this time it worked.